plugins/xpad/xpad.cpp

```diff
@@ -23,6 +23,10 @@
 #include "xpad.h"
 #include <VersionHelpers.h>
 
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x0602 // Required for XInputEnable definition
```

Comment:
Yes, I see it now. 
But shouldn't it.. I dunno, check for Windows SDK before setting that?